ic-os/guestos/context/docker-base.prod

```diff
@@ -1 +1 @@
-ghcr.io/dfinity/guestos-base@sha256:0a485d2d033df5aa7efd4ccbaad432c29d18f0f165cdcbeed5d49790c0d16340
+ghcr.io/dfinity/guestos-base@sha256:f27980807c63b88ce53d691df6e02df8385af074fcccadd1ca43d4ebb618dba6
```

Comment:
This contains changes for the GuestOS 24.04 upgrade, we'll keep this in an RC for the initial rollout